pom.xml

```diff
@@ -61,10 +61,9 @@ Portions Copyright (c) 2018, 2020, Chris Fraire <cfraire@me.com>.
 
     <properties>
         <lucene.version>9.7.0</lucene.version>
-        <mavenjavadocplugin.version>3.2.0</mavenjavadocplugin.version>
-        <!-- The following changed syntax from e.g. 1.8 to 11. -->
-        <compileSource>11</compileSource>
-        <compileTarget>11</compileTarget>
+        <mavenjavadocplugin.version>3.6.0</mavenjavadocplugin.version>
+        <compileSource>17</compileSource>
+        <compileTarget>17</compileTarget>
```

Comment:
For the time being please keep the target at 11. I'd like to run the releases on Java 11 even though the compilation uses JDK 17.